cogs/ticket.py

```diff
@@ -0,0 +1,235 @@
+
+import disnake
+from disnake.ext import commands
+from datetime import datetime
+from core.utils import TicketCog

```

Comment:
there's no TicketCog in utils.py
you meant HelperCog ig?

Reply:
um yeah i didnt setup that